src/celestia/qt/QtAudioManager.cpp

```diff
@@ -76,7 +75,9 @@ void QtAudioManager::playAll() {
 }
 
 void QtAudioManager::pauseAll() {
-    cout << "pauseAll()\n";
+#ifndef NO_DEBUG
+    cerr << "pauseAll()\n";
```

Comment:
It's not an error message

Reply:
that's just a matter of preferences, i usually prefer to have debug messages on stderr, but i'll change if you wish.